packages/plugins/plugin-idfa/README.md

```diff
@@ -36,6 +36,32 @@ const segmentClient = createClient({
 segmentClient.add({ plugin: new IdfaPlugin() });
 ```
 
+## Customize IDFA Plugin Initialization
+
+To delay the `IDFA Plugin` initialization (ie. to avoid race condition with push notification prompt) implement the following: 
+
+```ts
+import { createClient } from '@segment/analytics-react-native';
+
+import { IdfaPlugin } from '@segment/analytics-react-native-plugin-idfa';
+
+const segmentClient = createClient({
+  writeKey: 'SEGMENT_KEY'
+});
+
+...
+
+//The IDFA Plugin supports an optional `enabled` boolean 
+//setting to false disables plugin on initialization
```

Comment:
A more true explanation would be that setting to `false` makes it not request automatically permissions, however it is super important to make it explicit that if they want IDFA to show up they absolutely need to call `.enable` (or the renamed function, see comment below). Else the IDFA won't retrieve the values for IDFA.

We should also add that by default it will request IDFA permissions right away.